src/main/java/net/earthcomputer/clientcommands/command/FovCommand.java

```diff
@@ -0,0 +1,37 @@
+package net.earthcomputer.clientcommands.command;
+
+import com.mojang.brigadier.CommandDispatcher;
+
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.server.command.ServerCommandSource;
+import net.minecraft.text.Text;
+import net.minecraft.text.TranslatableText;
+
+import static com.mojang.brigadier.arguments.DoubleArgumentType.*;
+import static net.earthcomputer.clientcommands.command.ClientCommandManager.*;
+import static net.minecraft.server.command.CommandManager.*;
+
+public class FovCommand {
+
+    public static void register(CommandDispatcher<ServerCommandSource> dispatcher) {
+        addClientSideCommand("cfov");
+
+        dispatcher.register(literal("cfov")
+            .then(argument("fov", doubleArg())
+                .executes(ctx -> setGamma(ctx.getSource(), getDouble(ctx, "fov"))))
+            .then(literal("normal")
+                .executes(ctx -> setGamma(ctx.getSource(), 70)))
+            .then(literal("quakePro")
+                .executes(ctx -> setGamma(ctx.getSource(), 110))));
+    }
+
+    private static int setGamma(ServerCommandSource source, double fov) {
```

Comment:
setFov

Reply:
Fixed in f2cde2e0ecb5c6bc39483b47a5b6153d8b0898fc